.circleci/config.yml

```diff
@@ -84,7 +85,8 @@ jobs:
             echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
             sanitized_branch=`echo $CIRCLE_BRANCH | sed 's:/:-:g'`
             tag_suffix=$(echo "$CIRCLE_SHA1" | cut -c 1-8)
-            LILY_IMAGE_TAG=${sanitized_branch}-${tag_suffix}-butterflynet make docker-butterflynet-push
+            tag_date=$(date +%Y%m%d%H%M)
+            make docker-butterflynet-push LILY_IMAGE_TAG=${sanitized_branch}-${tag_date}-${tag_suffix}-butterflynet
```

Comment:
This adds the date to the butterflynet docker build tag. (should we eventually need it)